docs/UserGuide.md

```diff
@@ -296,7 +300,18 @@ ModBook data is saved as a JSON file `[JAR file location]/data/modbook.json`. Ad
 
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ModBook home folder.
 
-## Date and Time Formats
+## Day, Date and Time Formats
```

Comment:
maybe we should order the date format table to be above the time format table to correspond to the order in the heading